### DISCUSSION TITLE
Wake velocity deficit outputs for Curl wake model in FAST.Farm

### DISCUSSION THREAD

**Discussion by yisehakabreham** [OpenFAST/openfast#1492] [General]
Greetings,

The axial and radial wake velocity deficits, "WkDfVxTaNbDg" and "WkDfVrTaNbDg", are possible outputs in the Polar wake model.

But, they are not available in the curl model. I know it uses the cartesian projection, but how can I generate wake velocity deficits on downstream distances in the (X,Y,Z)?

The FAST.Farm cancels these outputs as follows:

![curl_model_vel](https://user-images.githubusercontent.com/93268626/225220524-94c3005d-d280-4d34-97df-66a2aa21791e.PNG)

**Comment by ebranlard** [comment]
Hi @yisehakabreham, 
Due to the fact that the wake is asymmetric, these outputs are indeed not available. Instead, you could set `OutAllPlanes` to True. 
https://openfast.readthedocs.io/en/dev/source/user/fast.farm/InputFiles.html#wind-turbines
This will write all the wake planes in VTK format. You will get similar outputs: the "raw" deficits (without the ambient flow) from these planes , and the eddy viscosity. Note that this is a lot of data and will slow down the simulation. The planes will be located in the meandering frame of reference and not in the absolute frame, which might be harder to interpret. 

If you are looking for the total wake deficit (i.e. including the ambient flow), then you can use  `OutDisWindYZ` which allows to write the velocity field at different X locations.  This is probably the most useful output.

**Reply by yisehakabreham** [reply]
@ebranlard 

Thanks for clearing it up. I will use VTK outputs under Visualization as you said. 

Yisehak A.
